src/components/CommitteeTools/components/Decrypt/index.scss

```diff
@@ -1,7 +1,39 @@
-@import "../../../../styles/constants.scss";
+@import "styles/constants.scss";
 
-// .keypair-list {
+.decrypt-wrapper {
+  width: 100%;
+  margin-bottom: $spacing-4;
+
+  &__textbox {
+    &-container {
+      margin-bottom: $spacing-3;
+    }
+    &-title {
+      margin-bottom: $spacing-3;
+    }
+  }
+}
+
+.selected-key {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: $spacing-3;
+
+  .box-with-copy {
+    align-items: center;
+    margin-right: 5px;
+    display: flex;
+    flex-direction: row;
+    padding: 3px;
+    flex: 1;
+    border: 1px solid #8afcfd;
```

Comment:
Please use color from the variables, not hardcoded.

Reply:
OK sure